content/techniques/queues.md

```diff
@@ -194,6 +194,25 @@ const job = await this.audioQueue.add(
 );
 ```
 
+#### Providers
+
+Whether the processor needs to depend on a request scoped provider or be explicitly registered as a scoped processor, as follows:
+
+```typescript
+@Processor({
+  name: 'audio',
+  scope: Scope.REQUEST,
+})
+```
+
+and then you can inject the JOB_REF via the constructor.
+
+```typescript
+constructor(@Inject(JOB_REF) jobRef: Job) {
+   console.log(jobRef);
+}
+```
+
```

Comment:
I assume `JOB_REF` can be imported from `@nestjs/bull`, can we add the hint for that?

Reply:
is there also at the beginning of the article you say to repeat it?

Reply:
I see information about jobs, but nothing about the `JOB_REF` constant. Can you point that out to me?

Reply:
I got confused :-), such a thing could also:

``` import { JOB_REF } from '@nestjs/bull' ```

Reply:
That should be fine.